viewing_party/party.py

```diff
@@ -1,23 +1,206 @@
 # ------------- WAVE 1 --------------------
 
+
 def create_movie(title, genre, rating):
-    pass
+
+   
+    movie = {
+        "title":title,
+        "genre":genre,
+        "rating":rating
+    }
+    #returns none if title or movie or rating is falsy
+    if title == None or genre == None or rating == None:

```

Comment:
This works, though it is better practice to use `is` to check for None values. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation why! It gets into how python objects' equality is defined and how it can be overridden, something we'll cover in OOP.